terraform/workspaces/infra/resource-group.tf

```diff
@@ -7,26 +7,3 @@ resource "random_string" "app" {
   lower   = true
   upper   = false
 }
-
-resource "aws_resourcegroups_group" "app" {
-  name        = local.workspace
-  description = "Resources in Paragon enterprise workspace ${local.workspace}"
-
-  resource_query {
-    query = <<JSON
-{
-  "ResourceTypeFilters": ["AWS::AllSupported"],
-  "TagFilters": [
-    {
-      "Key": "Workspace",
-      "Values": ["${local.workspace}"]
-    },
-    {
-      "Key": "Environment",
-      "Values": ["${local.environment}"]
-    }
-  ]
-}
-JSON
-  }
-}
```

Comment:
I found this to be really helpful to find all resources associated with an installation, particularly if there's Terraform state drift. In multi-tenant environments, like our internal sandbox enterprise account used for testing, we sometimes have multiple installations deployed and this is useful.

Still approving, but it'd be ideal to keep this.